### PR TITLE
MM-36296 generalize terminology

### PR DESCRIPTION
**Pull request by lieut-data**
#### Summary
`s/incident/playbook run/`, and then some.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-36296

#### Checklist
- [x] Telemetry updated
- [ ] ~~Gated by experimental feature flag~~
- [x] Unit tests updated

### CODE REVIEW

File: server/api/telemetry.go [hunk 1]
```diff
@@ -17,31 +17,31 @@ import (
...
 
 	telemetryRouter := router.PathPrefix("/telemetry").Subrouter()
 
 	startTrialRouter := telemetryRouter.PathPrefix("/start-trial").Subrouter()
 	startTrialRouter.HandleFunc("", handler.startTrial).Methods(http.MethodPost)
 
-	incidentTelemetryRouterAuthorized := telemetryRouter.PathPrefix("/incident").Subrouter()
-	incidentTelemetryRouterAuthorized.Use(handler.checkViewPermissions)
-	incidentTelemetryRouterAuthorized.HandleFunc("/{id:[A-Za-z0-9]+}", handler.telemetryForIncident).Methods(http.MethodPost)
+	playbookRunTelemetryRouterAuthorized := telemetryRouter.PathPrefix("/run").Subrouter()
```
Comment (lieut-data) [inline server/api/telemetry.go:42]: Using `/run` in URLs for brevity.

File: server/app/errors.go [hunk 2]
```diff
@@ -2,23 +2,23 @@ package app
...
 var ErrChannelDisplayNameInvalid = errors.New("channel name is invalid or too long")
 
-// ErrPermission is used to indicate a user does not have permissions
+// ErrPermission occurs when a user does not have permissions.
 var ErrPermission = errors.New("permissions error")
 
-// ErrIncidentNotActive is used to indicate trying to run a command on an incident that has ended.
-var ErrIncidentNotActive = errors.New("incident not active")
+// ErrPlaybookRunNotActive occurs when trying to run a command on a playbook run that has ended.
+var ErrPlaybookRunNotActive = errors.New("already ended")
```
Comment (lieut-data) [inline server/app/errors.go:15]: Simplifying instead of just replacing words here.

File: server/app/reminder.go [hunk 3]
```diff
@@ -12,64 +12,60 @@ import (
 	"github.com/pkg/errors"
 )
 
-type Reminder struct {
```
Comment (lieut-data) [inline server/app/reminder.go:15]: Entirely unused.

File: server/command/command.go [hunk 4]
```diff
@@ -21,23 +21,22 @@ import (
...
 
 const helpText = "###### Mattermost Incident Collaboration Plugin - Slash Command Help\n" +
-	"* `/incident start` - Start a new incident. \n" +
-	"* `/incident end` - Close the incident of that channel. \n" +
-	"* `/incident update` - Update the incident's status and (if enabled) post the status update to the broadcast channel. \n" +
-	"* `/incident check [checklist #] [item #]` - check/uncheck the checklist item. \n" +
-	"* `/incident checkadd [checklist #] [item text]` - add a checklist item. \n" +
-	"* `/incident checkremove [checklist #] [item #]` - remove a checklist item. \n" +
-	"* `/incident owner [@username]` - Show or change the current owner. \n" +
-	"* `/incident announce ~[channels]` - Announce the current incident in other channels. \n" +
```
Comment (lieut-data) [inline server/command/command.go:31]: Eliminating `announce`, given we have an automation for that.

**Review by lieut-data — COMMENTED** [review]
Proposing that we don't actually review the bulk of this -- apart from maybe the `sqlStore` changes -- and I'll promise to fix up anything that's broken after the fact. I've added a few comments, but not everything.

**Review by crspeller — APPROVED** [review]
Agreed. No need to spend the time looking though every change here. Just did a skim of the more critical portions and didn't see anything worrying. 

Awesome work! This must have been a bit tedious.

**Comment by lieut-data** [comment]
Thanks, @crspeller -- I'll wait for https://github.com/mattermost/mattermost-plugin-incident-collaboration/pull/602 and https://github.com/mattermost/mattermost-plugin-incident-collaboration/pull/601, resolve conflicts, and then merge this.

**Pull request merged**